lib/features/images/presentation/cubit/image_cubit.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:io';
+import 'package:fiura_ecosystem/features/images/presentation/cubit/image_state.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../../domain/repositories/image_repository.dart';
+
+class ImageCubit extends Cubit<ImageState> {
+  ImageCubit(this._imageRepository) : super(const Initial());
+
+  final ImageRepository _imageRepository;
+
+  void imagePicker() async {
+    Map<String, dynamic> result;
+
+    final File? image = await _imageRepository.imagePicker();
```

Comment:
call this repo where is needed there is no need of a new cubit just for handling images